src/components/css/LeafletMap.css

```diff
@@ -56,3 +56,11 @@
 .popup__text__center ion-text {
   margin-left: 1em;
 }
+
+/* override styling of leafletjs elements */
+
+.leaflet-control-zoom > a {
```

Comment:
The buttons are already looking a lot better, right now you cannot see that the buttons get disabled when you have zoomed in/ out fully.

After a bit of playing around I think this will fix the problem.
```css
.leaflet-control-zoom > a {
  background-color: var(--ion-color-light, var(--ion-background-color, #fff)) !important;
  border-color: var(--ion-color-light-tint) !important;
  color: var(--ion-color-dark) !important;
}

.leaflet-control-zoom > a.leaflet-disabled {
  color: var(--ion-color-medium) !important;
}
```

Reply:
ahah I didn't even know it did that ngl. I'll fix it

Reply:
Done. Also added hover state and changed other small things. Doesn't look as good as I want it to, but it's fine for now